vtex-apps/my-account
# Feature/idcs 90/show my authentication0.x

#### What did you change? \*

Added `Authentication` toggle to app configurations to set tab visibility 

![image](https://user-images.githubusercontent.com/22064061/112517053-a4794380-8d76-11eb-9b05-8270d96a408f.png)

#### Why? \*

The `Authentication` tab is a new section of MyAccount that will soon be present in all accounts. We're creating this visibility toggle, for now, to let our customers customize this UI before enabling it.

![image](https://user-images.githubusercontent.com/22064061/112519004-b4922280-8d78-11eb-9ece-d10db6441870.png)

#### How to test it? \*

Go [here](https://rafaprtest--recorrenciacharlie.myvtex.com/admin/apps/vtex.my-account@0.34.0/setup/) to access the visibility toggle.

Then go [here](https://recorrenciacharlie.vtexcommercestable.com.br/_secure/account?workspace=rafaprtest#/profile) before and after enabling `Authentication` visibility to check the presence of that section in the left-side menu.

#### Types of changes \*
